api-reference/_hidden/GridBaseColumn/lookup/dataSource.md

```diff
@@ -25,7 +25,7 @@ This option accepts one of the following:
 - **Array of objects or primitives**       
 A JavaScript array that contains plain objects or primitives.
 
-- [**DataSource configuration object**](/api-reference/30%20Data%20Layer/DataSource/1%20Configuration '/Documentation/ApiReference/Data_Layer/DataSource/Configuration/')       
+- [**DataSource configuration object**](/Documentation/Guide/Widgets/DataGrid/How_To/Bind_a_Lookup_Column_to_a_Custom_Data_Source/)       
```

Comment:
This link is unexpected under the caption DataSource configuration object. Yes. it doesn't explain how to use the object, we don't want to confuse our readers. Let's keep the old link here, but change the description a bit:

> A **DataSource** configuration object. You can find a code example for this use case in the following help topic: [Bind a Lookup Column to a Custom Data Source](). For more information about the **DataSource** and DevExtreme Data Layer, refer to the [Data Layer](/concepts/70%20Data%20Binding/5%20Data%20Layer '/Documentation/Guide/Data_Binding/Data_Layer/') article.